Makefile

```diff
@@ -229,10 +234,8 @@ release: ## 'make build-services' with cargo --release
 .PHONY: zip
 zip: build-aws ## Generate zips for deploying to AWS (src/js/grapl-cdk/zips/)
 	docker-compose -f docker-compose.zips.yml up
-
-.PHONY: deploy
-deploy: zip ## CDK deploy to AWS
-	src/js/grapl-cdk/deploy_all.sh
+	./pants filter --filter-target-type=python_awslambda :: | xargs ./pants package
```

Comment:
While we transition, it might be more flexible to have a distinct Makefile target for building lambdas via Pants, and then have a top-level target call both it and the old `docker-compose`-based workflow.

Reply:
done

Reply:
I think I'm going to spawn a separate ticket to track this, if that's alright.

Reply:
https://github.com/grapl-security/issue-tracker/issues/347